File: Problem Solving/C++/compare_the_triplets.cpp
#include<iostream>
using namespace std;
int main()
{
    int alice[3], bob[3], a = 0, b = 0;
    for(int i=0; i<3; i++)
        cin >> alice[i];
    for(int j=0; j<3; j++)
        cin >> bob[j];
    for(int k=0; k<3; k++)
    {
        if (alice[k] > bob[k])
            a++;
        if (alice[k] < bob[k])
            b++;
    }
    cout << a << " " << b;
    return 0;
}

File: Problem Solving/Python/compare_the_triplets.py
alice = list(map(int, input().split()))
bob = list(map(int, input().split()))
a = b = 0
for k in range(3):
    if alice[k] > bob[k]:
        a += 1
    if alice[k] < bob[k]:
        b += 1
print(a,b)